cmd/reconcile.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+	"os"
+	path "sigs.k8s.io/obscli/path"
+	"sigs.k8s.io/obscli/types"
+	"sigs.k8s.io/release-sdk/obs"
+	"sigs.k8s.io/yaml"
+)
+
+type Options struct {
+	ManifestPath string
+	OBSClient    *obs.OBS
+}
+
+type Info struct {
+	Username string
+	Password string
+	APIURL   string
+}
+
+func Reconcile() *cobra.Command {
+	ctx := context.Background()
+	opts := &Options{}
+	info := &Info{
```

Comment:
```suggestion
	info := &Info{
```
I've created an additional struct here to pass it to the New function of the library that helps creating the default client. Is there a way we can try to pass the information without making the struct again in the CLI Tool?

Reply:
You can instead use the Options struct from the `obs` package directly, e.g. `info := &obs.Options{`. Nit: I'd rename `info` to something like `obsOpts`

Reply:
done!